jsf-showcase-webapp/src/main/java/com/liferay/faces/showcase/converter/CountryConverter.java

```diff
@@ -65,14 +65,20 @@ public String getAsString(FacesContext facesContext, UIComponent uiComponent, Ob
 
 	protected Map<Long, Country> getCountryMap(FacesContext facesContext) {
 
+		// First check without locking (not yet thread-safe)
 		if (countryMap == null) {
 
 			synchronized (this) {
```

Comment:
@ngriffin7a, fixed here: https://github.com/liferay/liferay-faces-showcase/pull/87

For details see here: https://github.com/liferay/liferay-faces-bridge-impl/pull/283#discussion_r132963584